docs/release-notes/2.31.0.md

```diff
@@ -0,0 +1,174 @@
+
+We have upgraded Grafana to version 9.1 in PMM 2.31.0. With this upgrade, you can get a simplified user interface, enhanced visualizations, and a default unified Alerting experience. For more information, see What’s new in Grafana.
+ 
+The following features have been introduced with PMM 2.31.0:
+ 
+- **Expandable navbar (main menu)**
+ 
+    For an enhanced user experience and better accessibility, the main menu has been expanded with crucial dashbaords such as system, MongoDB, MySQL, PostgreSQL, ProxySQL, and HAProxy. Thus, these dashbaords cab be accessed upfront which makes it really conveneinet for the users.
+ 
+- **Prometheus Query Builder**
+
+    A new UI query builder has been introduced, by which you can write Prometheus queries effortlessly. Prometheus queries can be complex to grasp, but with this Grafana release in PMM, it becomes easier. 
+
+- **Improvements to dashboard search**
+    
+    The dashboards can now be searched by panel title as well as dashboard title. Also, the overall performance has been enhanced as querying and sorting have become much faster.
+
+
+### General availability of Percona Alerting
+
+With this release, we are streamlining the PMM alert setup process with an overhauled, unified alerting system based on Grafana. 
+
+All previous Integrated Alerting and Grafana Alerting functionality are now consolidated in a single pane of glass on the **Alerting** page. From here, you can configure, create and monitor alerts based on Percona or Grafana templates. 
+
+The **Alert Rules** tab has also evolved into a more efficient interface with an added layer for simplifying complex Grafana rules. You’ll find that the new **Percona templated alert** option here offers the same functionality available in the old Integrated Alerting, but uses a friendlier interface, powered by Grafana’s advanced alerting capabilities. 
+
+As an important and generally useful feature, this new Alerting feature is now enabled by default and ready to use in production! 
+
+### Deprecated Integrated Alerting
+The new Percona Alerting feature fully replaces the old Integrated Alerting in previous PMM versions. The new alerting brings full feature parity with Integrated Alerting, along with additional benefits like Grafana-based alert rules and a unified alerting command center. 
+
+If you have been using Integrated Alerting in the previous PMM version, your custom alert rule templates will be automatically migrated to PMM 2.31. After upgrading to this new version, you will find all your alert templates under **Alerting > Alert Templates**. 
+These templates are also available for creating new alert rules using the new Percona Templated Alerts option on the Alert Rules tab. 
+
+However, custom alert rules created with Integrated Alerting are not automatically migrated to Percona Alerting. After upgrading, make sure to manually migrate any custom alert rules that you want to transfer to PMM 2.31 using the script that will be provided soon. 
```

Comment:
Just `alert rules`, because there is no other rules.
```suggestion
However, alert rules created with Integrated Alerting are not automatically migrated to Percona Alerting. After upgrading, make sure to manually migrate any custom alert rules that you want to transfer to PMM 2.31 using the script that will be provided soon. 
```